src/ui/components/orientation_arrows.c

```diff
@@ -61,18 +62,21 @@ static void _render(component_t* component)
     int16_t position = data->screen_count / SCALE;
     if (position < COUNT_CHANGE_DIRECTION) {
         // Horizontal motion
+        qtouch_force_calibrate();
```

Comment:
instead of calling it in each if clause, why not just at the top of the render function?

Reply:
I agree with Marko that would be cleaner.

Reply:
I misunderstood how many times it would be called there before. My goal at that time was to only call calibrate at the end of the animation. However this is not needed anymore.